Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -742,6 +749,18 @@ void Wiimote::Update()
 			}
 		}
 
+		const ControlState sideways_toggle_pressed = m_hotkeys->controls[0]->control_ref->State();
+		const ControlState upright_toggle_pressed = m_hotkeys->controls[1]->control_ref->State();
+
+		if (sideways_toggle_pressed > s_sideways_toggle_last)
+			m_options->settings[1]->value = !(m_options->settings[1]->value);
```

Comment:
The checkboxes remain the same. If background input is on (and a game is running while the settings window is open), it'll change the layout.  Should I change the `if`s instead?